<launch>
  <arg name="ticks_per_meter" value="10000" />
  <arg name="wheel_separation" value="0.2" />
  <arg name="ppr" value="900"/>
  <arg name="d_wheel" value="0.06"/>
  <arg name="wheel_base" value="0.11"/>
  <arg name="urdf_file" default="$(find openbase)/urdf/robot.xacro" />
  <param name="robot_description" command="$(find xacro)/xacro --inorder $(arg urdf_file)" />

  <node name="node_odometry" pkg="openbase" type="node_odometry.py" output="screen"/>
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen">
    <param name="port" value="tcp"/>
  </node>
  <node name="teleop" pkg="openbase" type="node_teleop.py" output="screen"/>
  <node name="node_autonomous" pkg="openbase" type="node_autonomous.py" output="screen"/>

</launch>
